File: CSharp/Applications/AvaloniaLog4NetNoDI/DataStores/MainControlsDataStore.cs
using LogViewer.Core;
using LogDataStore = LogViewer.Avalonia.Logging.LogDataStore;

namespace AvaloniaLog4NetNoDI.DataStores;

// Application-wide shared instance of the LogDataStore logging entries
public static class MainControlsDataStore
{
    public static ILogDataStore DataStore { get; } = new LogDataStore();
}

File: VB/Applications/AvaloniaNlogVBNoDI/DataStores/MainControlsDataStore.vb
Imports LogViewerVB.Core
Imports LogDataStore = LogViewerVB.Avalonia.Logging.LogDataStore

Namespace DataStores

    ' Application-wide shared instance of the LogDataStore logging entries
    Public Module MainControlsDataStore

        Public ReadOnly Property DataStore As ILogDataStore = New LogDataStore()

    End Module

End Namespace